src/core/document/json.pl

```diff
@@ -704,11 +731,34 @@
     value_expand_list(Vs, DB, Context, Elt_Type, Captures_In_2, Exs, Dependencies_3, Captures_Out),
     append([Dependencies_1, Dependencies_2, Dependencies_3], Dependencies).
 
+value_expand_table([], _DB, _Context, _Elt_Type, Captures, [], [], Captures).
+value_expand_table([Value|Vs], DB, Context, Elt_Type, Captures_In, [Expanded|Exs], Dependencies, Captures_Out) :-
+    value_expand_list(Value, DB, Context, Elt_Type, Captures_In, Expanded, Dependencies_1, Captures_1),
+    value_expand_table(Vs, DB, Context, Elt_Type, Captures_1, Exs, Dependencies_2, Captures_Out),
+    append([Dependencies_1, Dependencies_2], Dependencies).
+
 % Unit type expansion
 context_value_expand(_,_,[],json{},Captures,[],[],Captures) :-
     !.
 context_value_expand(_,_,null,_,Captures,null,[],Captures) :-
     !.
+context_value_expand(DB,Context,Value,Expansion,Captures_In,V,Dependencies,Captures_Out) :-
+    get_dict('@container', Expansion, "@table"),
+    !,
+    % Container type
+    get_dict('@type', Expansion, Elt_Type),
+    (   is_list(Value)
+    ->  Value_List = Value
+    ;   string(Value)
+    ->  Value_List = [Value]
+    ;   get_dict('@value',Value,Value_List)
+    ->  true
```

Comment:
here too - is this a copy of the list code? are we sure that just works for tables?